src/styleProps/displayProps.ts

```diff
@@ -10,12 +11,12 @@ const supportedDisplayValues = [
 	'none',
 ];
 
-const overflowStylesMap = {
+const overflowStylesMap: {[key: string]: any} = {
```

Comment:
I think this `any` could just be `object`. But it would also be valid for it to return a string or a template literal, i.e. basically anything that can be interpolated into a styled component. Maybe `@types/styled-components` comes with a type for that?

Reply:
ideally we would do something like `const overflowStylesMap: {[key: string]: typeof ellipsis | typeof wrap}` but this leads to other typing problems for `displayPropsDef`. If you run it locally you will see what i mean. I think the fix here is cleaning up our types in multiple other places , but for now `any` allows us to make progress until we can come back to that.